erizo_controller/erizoClient/src/Connection.js

```diff
@@ -115,21 +121,35 @@ Erizo.GetUserMedia = function (config, callback, error) {
                         }
                         navigator.getMedia(theConfig,callback,error);
                     });
-                } catch (e){
+                } catch(e) {
                     L.Logger.debug('Screensharing plugin is not accessible ');
                     var theError = {code:'no_plugin_present'};
                     error(theError);
                     return;
                 }
                 break;
+
             default:
                 L.Logger.error('This browser does not support ScreenSharing');
         }
     } else {
-      if (typeof module !== 'undefined' && module.exports) {
-        L.Logger.error('Video/audio streams not supported in erizofc yet');
-      } else {
-        navigator.getMedia(config, callback, error);
-      }
+        if (typeof module !== 'undefined' && module.exports) {
+            L.Logger.error('Video/audio streams not supported in erizofc yet');
+        } else {
+            if (config.video && Erizo.getBrowser() == 'mozilla') {
```

Comment:
I believe the jshint is not smart enough. Using '==' for 'mozilla' is completely safe.